main.tf

```diff
@@ -25,6 +25,8 @@ locals {
   validate_cos_id_input = !var.create_cos_instance && var.existing_cos_instance_id == null ? tobool("If var.create_cos_instance is false, then provide a value for var.existing_cos_instance_id to create buckets") : true
   # tflint-ignore: terraform_unused_declarations
   validate_kp_guid_input = var.encryption_enabled && var.create_cos_instance && var.existing_key_protect_instance_guid == null ? tobool("A value must be passed for var.existing_key_protect_instance_guid when var.create_cos_instance and var.encryption_enabled is true.") : true
+  # tflint-ignore: terraform_unused_declarations
+  validate_cross_region_location_inputs = var.cross_region_location_enabled && var.cross_region_location == null ? tobool("If var.cross_region_location_enabled is true, then provide value for var.cross_region_location") : true
```

Comment:
I think we should update this to:

`validate_cross_region_location_inputs = var.create_cos_bucket && ((var.cross_region_location == null && var.region == null) || (var.cross_region_location != null && var.region != null)) ? tobool("If var.create_cos_bucket is true, then value needs to be provided for var.cross_region_location or var.region, but not both") : true`